[PATCH 6/6] Fix STRICT_ANSI vs POSIX
* libltdl/config/ltmain.m4sh (func_mode_link): Also check for _POSIX
as well as __STRICT_ANSI__ to avoid re-definitions.
---
 libltdl/config/ltmain.m4sh |    4 +++-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libltdl/config/ltmain.m4sh b/libltdl/config/ltmain.m4sh
index af46cb8..244bb5b 100644
--- a/libltdl/config/ltmain.m4sh
+++ b/libltdl/config/ltmain.m4sh
@@ -3382,7 +3382,7 @@
 
 /* declarations of non-ANSI functions */
 #if defined(__MINGW32__)
-# ifdef __STRICT_ANSI__
+# if defined(__STRICT_ANSI__) && !defined(__MINGW64_VERSION_MAJOR) || defined(_POSIX_)
 int _putenv (const char *);
 # endif
 #elif defined(__CYGWIN__)
-- 
1.7.0.2.msysgit.0